src/logic/safe/store/selectors/gatewayTransactions.ts

```diff
@@ -13,6 +12,9 @@ import { currentChainId } from 'src/logic/config/store/selectors'
 import { createHashBasedSelector } from 'src/logic/safe/store/selectors/utils'
 import { AppReduxState } from 'src/store'
 import { extractSafeAddress } from 'src/routes/routes'
+import { currentSafeNonce } from 'src/logic/safe/store/selectors/index'
+
+const BATCH_LIMIT = 10
```

Comment:
Is the limit defined by the multi-send contract? Or just arbitrary?

Reply:
> 10 could put a strain on the backend too if you were to request transaction details for each individual one.

Reply:
Gotcha, thanks